src/control/server/mgmt_system_test.go

```diff
@@ -1795,12 +1800,27 @@ func TestServer_MgmtSvc_SystemExclude(t *testing.T) {
 			if tc.req != nil && tc.req.Sys == "" {
 				tc.req.Sys = build.DefaultSystemName
 			}
+
+			startMapVer, err := svc.sysdb.CurMapVersion()
+			if err != nil {
+				t.Fatalf("startMapVer CurMapVersion() failed\n")
+				return
+			}
 			gotResp, gotAPIErr := svc.SystemExclude(ctx, tc.req)
 			test.CmpErr(t, tc.expAPIErr, gotAPIErr)
 			if tc.expAPIErr != nil {
 				return
 			}
 
+			endMapVer, err := svc.sysdb.CurMapVersion()
+			if err != nil {
+				t.Fatalf("endMapVer CurMapVersion() failed\n")
+				return
+			}
+
+			if endMapVer-startMapVer != tc.expVerIncr {
+				t.Fatalf("unexpected sysdb map version (want %d, got %d)\n", startMapVer+tc.expVerIncr, endMapVer)
+			}
```

Comment:
As I mentioned previously, the group update is an asynchronous operation. I'm a little concerned that the test as implemented here may be flaky, as I think there is a race between the group update and the check for the version increment.

I recommend just putting this section inside a simple for loop, and you can break out if the version has been incremented as expected. If for some reason the version never increments, then the test will time out after 30s. There's no danger of an infinite loop causing problems for other tests.

The other suggestion I have is that checking for a specific version may wind up making the test more brittle than necessary. All we really care about is that the version was incremented after the group update request. The amount by which it was incremented is an implementation detail that could change at some point.